db/schema.sql

```diff
@@ -1,4 +1,37 @@
-DROP DATABASE IF EXISTS byebyefresh_db;
+DROP DATABSE IF EXISTS byebyefresh_db;
 
 CREATE DATABASE byebyefresh_db;
 
+-- Uses the byebyefresh_db database
+USE byebyefresh_db;
+
+-- Creates the Recipe table
+CREATE TABLE Recipe (
+  id INT PRIMARY KEY AUTO_INCREMENT,
+  name VARCHAR(255) NOT NULL
+);
+
+-- Creates the Ingredient table
+CREATE TABLE Ingredient (
+  id INT PRIMARY KEY AUTO_INCREMENT,
+  name VARCHAR(255) NOT NULL
+);
+
+-- Creates the RecipeIngredient table with multiple foreign keys to allow for many-to-many relationship
+CREATE TABLE RecipeIngredient (
+  id INT PRIMARY KEY AUTO_INCREMENT,
+  recipeId INT NOT NULL,
+  ingredientId INT NOT NULL,
+  FOREIGN KEY (recipeId) REFERENCES Recipe(id),
+  FOREIGN KEY (ingredientId) REFERENCES Ingredient(id)
+);
+
+-- I believe we need this to handle the user info?
```

Comment:
@JanicaJensen I created the user table in User.js , so I won't be needing this here. So I'm deleting this user table from here, since its getting conflicted.